src/plugins/StatusBar/StatusBar.js

```diff
@@ -106,24 +106,59 @@ module.exports = (props) => {
   }
 
   const width = typeof progressValue === 'number' ? progressValue : 100
+  const isHidden = (uploadState === STATE_WAITING && props.hideUploadButton) ||
+    (uploadState === STATE_WAITING && !props.newFiles > 0)
 
   return html`
     <div class="UppyStatusBar is-${uploadState}"
-                aria-hidden="${uploadState === STATE_WAITING}"
-                title="">
-      <progress style="display: none;" min="0" max="100" value=${progressValue}></progress>
+         aria-hidden="${isHidden}">
       <div class="UppyStatusBar-progress ${progressMode ? `is-${progressMode}` : ''}"
-           style="width: ${width}%"></div>
+           style="width: ${width}%"
+           role="progressbar"
+           aria-valuemin="0"
+           aria-valuemax="100"
+           aria-valuenow="${progressValue}"></div>
```

Comment:
here `progressValue` can be null or undefined if we are processing and the processing progress is `'indeterminate'`, we should not add the valuenow attribute in that case